### ISSUE TITLE
[FAKE 5.8.4] PATH environment variable overwritten in `DotNet.build`

### ISSUE THREAD

**Issue by SteveGilham** [fsharp/FAKE#2134]
### Description

I have a program pre-build step which invokes `pwsh`, which is on the PATH; this step works fine in Visual Studio, and when built through a FAKE target that invokes `DotNet.build` on the relevant solution in FAKE <= 5.7.2; but on upgrading to 5.8.4 this step fails with
```
'pwsh' is not recognized as an internal or external command,
  operable program or batch file.
```

### Repro steps

1. Create a .net core project with the target
```
  <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
    <Exec Command="echo %PATH%" />
  </Target>
```

2. build with Fake 5.8.4

### Expected behavior

the value of PATH is as it would be outside the Fake script, as was the case with Fake 5.7.2, and in particular to still include `C:\Program Files\PowerShell\6\`

### Actual behavior

In the build with Fake 5.8.4, this emits only
```
  C:\Program Files\dotnet;
```

### Known workarounds

Currently, I've just reverted to 5.7.2
It would be possible to do the same work within the FAKE script, at the cost of making this particular project require building through Fake and not the IDE

### Related information 

* Operating system -- WinX Home
* Branch --  NuGet packages for 5.8.4
* .NET Runtime, CoreCLR or Mono Version -- .net core 2.1.402
* Indications of severity -- this is a significant, and possibly unintentional, change in behaviour at a minor release
* Version of FAKE (4.X, 5.X) -- 5.8.4

**Comment by matthid** [comment]
Any chance for me to reproduce this?

**Comment by matthid** [comment]
Ah you say it reproduces with the minimal project file & usind dotnet build?

**Comment by SteveGilham** [comment]
Yes, nothing fancy is involved in the build; in particular no environment mapping.  If you have problems, then I'll do the cutting down of the real case.

**Comment by matthid** [comment]
@SteveGilham Should be fixed now with `5.8.5`

**Comment by SteveGilham** [comment]
@matthid -- Works like a charm.  Thanks once again!
